README.md

```diff
@@ -12,7 +12,7 @@ manifests, image indexes (multi-arch images) and custom artifacts
 * **Multi-arch images**, compiling and building multi-arch images with a single Bazel invocation
 
 In addition to Bazel rules, we offer many helpers for interacting with OCI
-artifacts under the `pkg` directory and a CLI tool for creating new OCI
+artifacts under the `go/pkg` directory and a CLI tool for creating new OCI
```

Comment:
May be worth noting here that this is also in its own Go module and what to "go get" if they do want to use the helpers.

Reply:
Good call, will add that momentarily.

Reply:
Done!